schemas/Clinical/index.js

```diff
@@ -31,7 +31,8 @@ const typeDefs = gql`
   type ClinicalRegistrationData @cost(complexity: 10) {
     id: ID
     creator: String
-
+    fileName: String
+    createdAt: String
```

Comment:
there's a `DateTime` scalar that should be used for date timestamps for consistency :)
See an example usage in: `schemas/Program/index.js`
Resolver for that scalar is in: `schemas/customScalars.js`